build-logic/settings.gradle.kts

```diff
@@ -1,4 +1,11 @@
 dependencyResolutionManagement {
+    // Configure repositories
+    // TODO: find mystery 'ivy' repository
```

Comment:
Currently is not possible to use `FAIL_ON_PROJECT_REPOS` with some K/N targets: https://youtrack.jetbrains.com/issue/KT-51379